src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeoException.scala

```diff
@@ -0,0 +1,15 @@
+package org.broadinstitute.dsde.workbench.leonardo.model
+
+import akka.http.scaladsl.model.StatusCode
+import org.broadinstitute.dsde.workbench.model.{ErrorReport, WorkbenchException, WorkbenchExceptionWithErrorReport}
+import org.broadinstitute.dsde.workbench.leonardo.errorReportSource
+import org.broadinstitute.dsde.workbench.model.ErrorReport._
+
+abstract class LeoException(message: String = null, cause: Throwable = null) extends WorkbenchException(message, cause) {
+  def toErrorReport(statusCode: StatusCode): WorkbenchExceptionWithErrorReport = {
+    Option(cause) match {
```

Comment:
this is a disgusting holdover from `WorkbenchException` in workbench-libs, which defines both message and cause as default-null.

Reply:
note also that we sanitise the stack trace (the empty `Seq()`) so users don't get to peek in our code.